sdk/go/src/sawtooth_sdk/messaging/stream.go

```diff
@@ -105,107 +164,132 @@ func (self *Stream) Respond(t validator_pb2.Message_MessageType, c []byte, corrI
 	}
 
 	self.outgoing <- request
+
+	return request.Response
 }
 
+// Setup the sockets and channels needed for communicating.
```

Comment:
Usage: Setup is a noun, "set up" is a verb/imperative.
Rephrase: "Set up the sockets..."